atc/db/job_factory.go

```diff
@@ -30,6 +35,156 @@ func NewJobFactory(conn Conn, lockFactory lock.LockFactory) JobFactory {
 	}
 }
 
+type SchedulerJobs []SchedulerJob
+
+type SchedulerJob struct {
+	Job
+	Resources     SchedulerResources
+	ResourceTypes atc.VersionedResourceTypes
+}
+
+type SchedulerResources []SchedulerResource
+
+type SchedulerResource struct {
+	Name   string
+	Type   string
+	Source atc.Source
+	Tags   atc.Tags
```

Comment:
🤔 Why are the tags necessary? I may be missing something. Just seems like the odd one out.

Reply:
Oh ya I don't think it's necessary, looks like we don't use it to construct the build plan!